state/azure/tablestorage/tablestorage.go

```diff
@@ -0,0 +1,206 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package tablestorage
+
+import (
+	"fmt"
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/dapr/components-contrib/state"
+	jsoniter "github.com/json-iterator/go"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"strings"
+)
+
+const (
+	fullmetadata        = "application/json;odata=fullmetadata"
+	daprDelim           = "__delim__"
```

Comment:
Delimiter has been changed to `||` because `||` is more readable than `__delim__` : https://github.com/dapr/dapr/pull/960/files

I would rename `daprDelim` to `keyDelimiter` because component-contrib is intended to be used for the other projects. (who knows someone would want to use component-contrib in their project :) )

Reply:
Yes, best to use `keyDelimiter`.

Reply:
Thanks for this. I was wondering if it would make sense to export this constant somewhere, because currently it's hardcoded in several places:

![image](https://user-images.githubusercontent.com/3155189/73060964-f67fba00-3e90-11ea-88a1-59aed31f7355.png)

Reply:
anyway, fixed in my code for now